src/hb-ot-shape-complex-vowel-constraints.cc

```diff
@@ -34,6 +34,9 @@ _hb_preprocess_text_vowel_constraints (const hb_ot_shape_plan_t *plan HB_UNUSED,
 				       hb_buffer_t              *buffer,
 				       hb_font_t                *font HB_UNUSED)
 {
+  if (unlikely (buffer->flags & HB_BUFFER_FLAG_DO_NOT_INSERT_DOTTED_CIRCLE))
```

Comment:
This file is generated.  Please update the generator as well.

Also, the use of unlikely is not suggested in this case since it's not an exceptional case but a configuration.  Not a big deal.